bitrise.yml

```diff
@@ -0,0 +1,100 @@
+format_version: 4
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  # An example secret param, define it (A_SECRET_PARAM) in .bitrise.secrets.yml
+  - A_SECRET_PARAM: $A_SECRET_PARAM
+  # If you want to share this step into a StepLib
+  - BITRISE_STEP_ID: espresso-browserstack-official
+  - BITRISE_STEP_VERSION: "0.0.1"
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/browserstack/browserstack-bitrise-espresso-step.git
+  - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
+
+workflows:
+  test:
```

Comment:
we can remove test: right?

Reply:
resolved,
this workflow is used to trigger the test for this plugin on bitrise with sample inputs/params

Reply:
Discussed with @deepwired on the call, marking as resolved